### DISCUSSION TITLE
My local backend.ai enviorment provide a limited enviorment list.

### DISCUSSION THREAD

**Discussion by mindgitrwx** [lablup/contribution-academy-2021#78] [Q&A]
**Describe the problem**

My local backend.ai provides a limited enviorment, version list.
It only supports python environment, and  3.8/Ubuntu 18.04 version.
It seems no problem on my webUI. 

**To Reproduce Steps to reproduce the behavior:**
   
   Download install-dev.sh in backend.ai meta repository via wget command.
   Execute 
```
   python -m ai.backend.manager.server --debug 
   python -m ai.backend.agent.server --debug
   python -m ai.backend.storage.server --debug
   python -m ai.backend.web.server --debug
```
   Enter the address into your browser.  

`   http://127.0.0.1:8090/`

  Click a session Icon on the left frame and click a start button located on the right side.
**Expected behavior**

![image](https://user-images.githubusercontent.com/37054916/130332795-2603fd76-040a-45b6-96dd-82375bc1f308.png)

**Screenshots**

```
$ backend.ai run --rm --exec 'python single_minist.py' tf2 ./single_minist.py ./minist.py
∙ Session name prefix: pysdk-16f0ef3540
✘ [0] BackendAPIError(400, 'Bad Request', {'type': 'https://api.backend.ai/probs/invalid-api-params', 'title': 'Missing or invalid API parameters.', 'msg': 'Unknown image reference: tf2'})
```

![session](https://user-images.githubusercontent.com/37054916/130332491-2376243a-d1d5-4a31-bcfb-e70cf4edaa48.png)

Environment Description

OS: Ubuntu 18.04
App version: 21.09.0.dev0
Browser version: 91.01 (64bit)
Python version: 3.9.5
Current Shell: bash

**Comment by agatha197** [comment]
Pull the image what you want and try again. You can do that via webui or cli. you can see webui manual in [this link](https://console.docs.backend.ai/en/latest/admin_menu/admin_menu.html#manage-images)

**Reply by agatha197** [reply]
You'd better refer to [this](https://github.com/lablup/contribution-academy-2021/discussions/82) too.
